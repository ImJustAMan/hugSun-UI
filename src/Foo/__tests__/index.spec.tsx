import React from 'react';
import { mount } from 'enzyme';
import Foo from '..';

describe('Foo', () => {
  it('Foo is h1 tag', () => {
    const wrapper = mount(<Foo title="title" />);

    console.dir(wrapper.html());
    expect(wrapper).toBeTruthy();
  });
});
