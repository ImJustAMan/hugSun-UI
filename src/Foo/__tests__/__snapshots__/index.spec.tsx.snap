// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo Foo is h1 tag 1`] = `ReactWrapper {}`;

exports[`Foo is h1 tag`] = `
  <h1>
    title
  <h1>
`;
